### PR TITLE
F02 - Finalize Admin implementation

### PR DESCRIPTION
**Pull request by 1milosz2**
Please take a look on a testing that I recently added.

### CODE REVIEW

**Comment by 1milosz2** [comment]
Perfect

**Comment by krzysztofreczek** [comment]
Implementation of: https://github.com/krzysztofreczek/how-long/issues/5

File: how-long-admin/src/test/java/com/h8/howlong/admin/commands/impl/DeleteCommandTest.java [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package com.h8.howlong.admin.commands.impl;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class DeleteCommandTest {
```
Comment (krzysztofreczek) [inline how-long-admin/src/test/java/com/h8/howlong/admin/commands/impl/DeleteCommandTest.java]: This test is still to be implemented

File: how-long-admin/src/test/java/com/h8/howlong/admin/commands/impl/ListCommandTest.java [hunk 2]
```diff
@@ -0,0 +1,7 @@
+package com.h8.howlong.admin.commands.impl;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class ListCommandTest {
```
Comment (krzysztofreczek) [inline how-long-admin/src/test/java/com/h8/howlong/admin/commands/impl/ListCommandTest.java]: This test is still to be implemented

File: how-long-admin/src/test/java/com/h8/howlong/admin/commands/impl/UpdateEndCommandTest.java [hunk 3]
```diff
@@ -0,0 +1,7 @@
+package com.h8.howlong.admin.commands.impl;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class UpdateEndCommandTest {
```
Comment (krzysztofreczek) [inline how-long-admin/src/test/java/com/h8/howlong/admin/commands/impl/UpdateEndCommandTest.java]: This test is still to be implemented

File: how-long-admin/src/test/java/com/h8/howlong/admin/commands/impl/UpdateFullCommandTest.java [hunk 4]
```diff
@@ -0,0 +1,7 @@
+package com.h8.howlong.admin.commands.impl;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class UpdateFullCommandTest {
```
Comment (krzysztofreczek) [inline how-long-admin/src/test/java/com/h8/howlong/admin/commands/impl/UpdateFullCommandTest.java]: This test is still to be implemented

File: how-long-admin/src/test/java/com/h8/howlong/admin/commands/impl/UpdateStartCommandTest.java [hunk 5]
```diff
@@ -0,0 +1,7 @@
+package com.h8.howlong.admin.commands.impl;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class UpdateStartCommandTest {
```
Comment (krzysztofreczek) [inline how-long-admin/src/test/java/com/h8/howlong/admin/commands/impl/UpdateStartCommandTest.java]: This test is still to be implemented

File: how-long-admin/src/test/java/com/h8/howlong/admin/services/TimesheetManagementServiceTest.java [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+
+import com.h8.howlong.services.TimesheetContextService;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.ArgumentCaptor;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.*;
+
+public class TimesheetManagementServiceTest {
```
Comment (krzysztofreczek) [inline how-long-admin/src/test/java/com/h8/howlong/admin/services/TimesheetManagementServiceTest.java]: There are some cases missing in this test class

**Pull request merged**